ClubService.Domain/Model/Entity/Member.cs

```diff
@@ -108,6 +108,30 @@ public List<DomainEnvelope<IMemberDomainEvent>> ProcessMemberDeleteCommand()
         return [domainEnvelope];
     }
     
+    public List<DomainEnvelope<IMemberDomainEvent>> ProcessMemberChangeFullNameCommand(FullName fullName)
+    {
+        switch (Status)
+        {
+            case MemberStatus.ACTIVE:
+                var memberFullNameChangedEvent = new MemberFullNameChangedEvent(fullName);
```

Comment:
check if the provided fullname is different than the current one. Otherwise it doesn't make sense to create the event.